recipes/odm2rest/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.1.0a" %}
+
+package:
+  name: odm2rest
+  version: {{ version }}
+
+source:
+  # not useing the jinja variable for the version b/c the release is not pep440
+  # see https://www.python.org/dev/peps/pep-0440/#id25
+  url: https://github.com/ODM2/ODM2RESTfulWebServices/archive/v0.1.0-alpha.tar.gz
```

Comment:
When issuing a release you can use a local tag, that way versioneer will ensure a version number that is pep 440 compliant. (That makes out life easier b/c we don't need to keep track of two different forms of the version just for packaging.)

Reply:
@ocefpaf I created the tags. Are you saying it's not pep 440 compliant? I thought I was following the conventions we've been using (with your guidance). Can you suggest a tweak to the tag? Can we edit the release to change the tag? We can also delete the release (if that's possible) and issue a new one with a compliant tag.

Reply:
No need to delete it or tweak. I have a workaround in the recipe for that.

Did you make the tag manually or via the CLI? The issue with the current tag is the `-`. Please see https://www.python.org/dev/peps/pep-0440/#pre-releases for mor information on that.

Reply:
> No need to delete it or tweak. I have a workaround in the recipe for that.

Ok. But it really is no problem to edit/delete/recreate the release. It would not cause any disruptions to anyone.

> Did you make the tag manually or via the CLI? 

Manually.

> The issue with the current tag is the -. Please see https://www.python.org/dev/peps/pep-0440/#pre-releases for mor information on that.

Got it, thanks. I should've done better checking. For the next release -- and all other ODM2 software -- let's make sure we use compliant schemes. No sense in propagating less than ideal practices and build workarounds, when we can get this right.

Reply:
>  It would not cause any disruptions to anyone.

It does. The checksum changes when someones edit a releases.